### PR TITLE
add aesb2

### PR DESCRIPTION
**Pull request by benjub**
@wlammen : here is the simpler version of wl-ich-lem1 I mentioned in https://github.com/metamath/set.mm/pull/3347#discussion_r1285196705. I think that result is interesting in itself so deserves to be in Main. As for contributions, I put the two of us on equal footing (the order is alphabetic), but if you prefer, I can put two tags: "Contr. by WL" and "Revised by BJ".

Maybe you also have a proposal for a better label?

### CODE REVIEW

File: set.mm [hunk 1]
```diff
@@ -21197,6 +21197,17 @@ primitive connectives (no defined terms) on the right-hand side.
     dfsb1 ) BCEZBFZBDEZAGZUBAHZBIZHCDEZAGZUFAHZCIZHABDJACDJUAUCUGUEUIUAUBUFATUB
     UFKBBCDLMZNUDUHBCUAUBUFAUJOPQABDSACDSR $.
 
+  ${
+    $d v y $.
+    $( In the case of two successive substitutions for two always equal
+       variables, the second substitution has no effect.  (Contributed by BJ
+       and WL, 9-Aug-2023.) $)
+    aesb2 $p |-
```
Comment (wlammen) [inline set.mm:21205]: The ae standing for "all equal" is often added as a suffix, as in cbvaev, hbae for example,  The rest is fine from my POV.

Comment (benjub) [reply set.mm:21205]: Relabeled to sb2ae.

**Comment by benjub** [comment]
I also removed sbnf2OLD before the 1-year period since the proof is nearly identical to that of sbnf2 (just replace uses of sb8(e) with sb8(e)v and that's it):
https://us.metamath.org/mpeuni/sbnf2OLD.html
versus
https://us.metamath.org/mpeuni/sbnf2.html

**Comment by benjub** [comment]
Regenerate `discouraged` and remove https://us.metamath.org/mpeuni/sbelxOLD.html for the same reason: nearly identical to sbelx.

**Review by wlammen — APPROVED** [review]

**Pull request merged**